### PR TITLE
Updating databricks getting-started

### PR DESCRIPTION
**Pull request by sameerz**
Signed-off-by: Sameer Raheja <sraheja@nvidia.com>

Updating Databricks instructions on gh-pages.  Same changes as 0ff857d on branch-0.2.

### CODE REVIEW

**Comment by revans2** [comment]
build

File: docs/get-started/getting-started-with-rapids-accelerator-on-databricks.md [hunk 1]
```diff
@@ -6,48 +6,53 @@ parent: Getting-Started
...
+    * Apache Spark 3.0 running in DataBricks Runtime 7.0 ML with GPU 
     * AWS: 7.0 ML (includes Apache Spark 3.0.0, GPU, Scala 2.12)
     * Azure: 7.0 ML (GPU, Scala 2.12, Spark 3.0.0)
 
 The number of GPUs per node dictates the number of Spark executors that can run in that node.
 
 ## Start a Databricks Cluster
 Create a Databricks cluster by going to Clusters, then clicking “+ Create Cluster”. Ensure the cluster meets the prerequisites above by configuring it as follows:
-1. On AWS, make sure to use 7.0 ML (includes Apache Spark 3.0.0, GPU, Scala 2.12), or for Azure, choose 7.0 ML (GPU, Scala 2.12, Spark 3.0.0).
+1. On AWS, make sure to use 7.0 ML (GPU, Scala 2.12, Spark 3.0.0), or for Azure, choose 7.0 ML (GPU, Scala 2.12, Spark 3.0.0).
```
Comment (jlowe) [inline docs/get-started/getting-started-with-rapids-accelerator-on-databricks.md:20]:
```suggestion
1. On AWS and Azure, make sure to use 7.0 ML (GPU, Scala 2.12, Spark 3.0.0).
```

File: docs/get-started/getting-started-with-rapids-accelerator-on-databricks.md [hunk 2]
```diff
@@ -6,48 +6,53 @@ parent: Getting-Started
...
-Select the “Init Scripts” tab in the advanced options section, and paste the initialization script: `dbfs:/databricks/init_scripts/init.sh`, then click “Add”
+3. Ensure that the newly created init.sh script is present in the output from cell 2 and that the contents of the script are correct.
+4. Go back and edit your cluster to configure it to use the init script.  To do this, click the “Clusters” button on the left panel, then select your cluster.
+5. Click the “Edit” button, then navigate down to the “Advanced Options” section.  Select the “Init Scripts” tab in the advanced options section, and paste the initialization script: `dbfs:/databricks/init_scripts/init.sh`, then click “Add”. 
 
-![Init Script](../img/initscript.png)
+    ![Init Script](../img/initscript.png)
+
+6. Now select the “Spark” tab, and paste the following config options into the Spark Config section.  Change the config values based on the workers you choose.  See Apache Spark [configuration](https://spark.apache.org/docs/latest/configuration.html) and RAPIDS Accelerator for Apache Spark [descriptions](../configs) for each config.
```
Comment (jlowe) [inline docs/get-started/getting-started-with-rapids-accelerator-on-databricks.md:39]:
```suggestion
6. Now select the “Spark” tab, and paste the following config options into the Spark Config section.  Change the config values based on the workers you chose.  See Apache Spark [configuration](https://spark.apache.org/docs/latest/configuration.html) and RAPIDS Accelerator for Apache Spark [descriptions](../configs) for each config. 

```

**Comment by jlowe** [comment]
Ah nevermind, this is just sync'ing gh-pages.  We can address my comments in a followup on branch-0.2.

**Pull request merged**